src/components/cards/remote-cards/RemoteCard.tsx

```diff
@@ -204,12 +204,9 @@ export const RemoteCard: React.FC<RemoteCardProps> = ({ card = {} as TrimmedCard
 
                 {!imageIcon && imageUri && (
                   <Box
-                    as={ImgixImage}
-                    tintColor={accent}
-                    source={{ uri: imageUri }}
-                    resizeMode="cover"
+                    as={RemoteSvg}
```

Comment:
are we sure this is always going to be an svg? personally would prefer to use images here